File: java/web/src/main/java/web/template/entity/TreeFormNode.java
package web.template.entity;

import java.util.ArrayList;
import java.util.List;

public class TreeFormNode {
	
	/**
	 * 节点d
	 */
	private String id;

    /**
     * 节点名称
     */
	private String name;

    /**
     * 子节点
     */
	private List<TreeFormNode> children;

    public TreeFormNode()
    {
        children = new ArrayList<TreeFormNode>();
    }

	public String getId() {
		return id;
	}

	public void setId(String id) {
		this.id = id;
	}

	public String getName() {
		return name;
	}

	public void setName(String name) {
		this.name = name;
	}

	public List<TreeFormNode> getChildren() {
		return children;
	}

	public void setChildren(List<TreeFormNode> children) {
		this.children = children;
	}
}

File: c#/web/WebApplication1/Entity/Common/TreeFormNode.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Web;

namespace WebApplication1.Entity.Common
{
    /// <summary>
    /// 树表单节点
    /// </summary>
    public class TreeFormNode
    {
        /// <summary>
        /// 节点id
        /// </summary>
        public string id { set; get; }

        /// <summary>
        /// 节点标题
        /// </summary>
        public string name { set; get; }

        /// <summary>
        /// 树节点的子节点
        /// </summary>
        public List<TreeFormNode> children { set; get; }

        public TreeFormNode()
        {
            children = new List<TreeFormNode>();
        }
    }
}